samples/ChatSample/ChatSample.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="BuildBundlerMinifier" Version="2.4.337" />
```

Comment:
This package carries an [MSBuild target](https://github.com/madskristensen/BundlerMinifier/blob/master/src/BundlerMinifier/build/BuildBundlerMinifier.targets) that chains in automatically to "Build" and "Clean".